spec/features/access_etd_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.feature 'Access an Etd', js: false do
+  let(:etd)         { FactoryGirl.create(:public_etd, pdf: pdf) }
+  let(:pdf)         { FactoryGirl.create(:pdf_upload) }
+  let(:private_etd) { FactoryGirl.create(:etd, pdf: pdf) }
+
+  scenario 'searching submitted ETDs' do
+    visit '/'
+
+    fill_in 'Search Hyrax', with: etd.first_title
+    click_button 'Go'
+
+    within(:css, "li#document_#{etd.id}") do
+      expect(page).to have_content etd.first_title
+    end
+  end
+
```

Comment:
it feels like it might be good to have a test demonstrating the restricted etd does not appear in search results even when you search for the title.

Reply:
I started to write a test like this, but found that `restricted` items *do* appear in search results, so I opened #87.